scripts/storage/qcow2/managesnapshot.sh

```diff
@@ -187,6 +187,7 @@ backup_snapshot() {
   local snapshotname="$2"
   local destPath=$3
   local destName=$4
+  local forceShareFlag="-U"
```

Comment:
@davidjumani @shwstppr can you explain this change and how it is used - is this for the snapshot failure fix?

Reply:
@rhtyd  `qemu-img` with QEMU 2.1 is not able to read QCOW2 file when it is already have a read lock by another process (libvirt). Due to this, both Ubuntu and CentOS8 fail to snapshot a running VM.
For a running VM and volume snapshot, as per code (and my understanding), we create an external snapshot for VM using libvirt, then this script is used to read, convert (to a new file) and backup that snapshot to secondary storage using qemu-img. Since we are not making any change as such in original QCOW2 disk file, we can open it shared open (`-U`) flag which will prevent regression from CentOS7 or distros with older QEMU.
I've added checks in code avoid any failures if `-U` is not supported

Reply:
qemu 2.10 I believe, which also means when using qemu-kvm-ev 2.10+ on i.e. CentOS7 is fixed